File: RukaLjubavi.Api/Models/Benefiktor.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace RukaLjubavi.Api.Models
{
    public class Benefiktor : IEntity
    {
        [Key]
        public int Id { get; set; }
        public string NazivKompanije { get; set; }
        public string PDVbroj { get; set; }

        [ForeignKey(nameof(KorisnikId))]
        public Korisnik Korisnik { get; set; }
        public int KorisnikId { get; set; }
    }
}

File: RukaLjubavi.Android/app/src/main/java/com/fit/ba/rukaljubavi/Models/Benefiktor.kt
package com.fit.ba.rukaljubavi.Models

import java.io.Serializable

class Benefiktor(var id: Int,var korisnikId: Int,var nazivKompanije: String,var pdvbroj: String, var email: String,
                 var telefon: String, var adresa: String, var mjestoPrebivalista: String,var mjestoPrebivalistaId: Int ,var datumRegistracije: String, var BrojDonacija: Int) : Serializable {
}